PowerEditor/src/ScintillaComponent/ScintillaEditView.cpp

```diff
@@ -3235,41 +3229,70 @@ TCHAR * int2str(TCHAR *str, int strLen, int number, int base, int nbChiffre, boo
 			mask >>= 1;
 		}
 		str[i] = '\0';
-	}
+		// str is now leading zero padded
 
-	if (!isZeroLeading)
-	{
-		if (base == 2)
+		if (isSpaceLeading)
 		{
-			TCHAR *j = str;
-			for ( ; *j != '\0' ; ++j)
-				if (*j == '1')
+			// replace leading zeros with spaces
+			for (TCHAR* j = str; *j != '\0'; ++j)
+			{
+				if ((*j == '1') || (*(j + 1) == '\0'))
+				{
 					break;
-			wcscpy_s(str, strLen, j);
+				}
+				else
+				{
+					*j = ' ';
+				}
+			}
 		}
-		else
+		else if (!isZeroLeading)
 		{
-			// use sprintf or swprintf instead of wsprintf
-			// to make octal format work
-			swprintf(f, bufSize, TEXT("%%%s"), fStr);
-			swprintf(str, strLen, f, number);
+			// left-align within the field width, i.e. pad on right with space
+			
+			// first, remove leading zeros
+			for (TCHAR* j = str; *j != '\0'; ++j)

```

Comment:
Please add the braces around the whole instruction of condition `if (*j == '1' || *(j + 1) == '\0')` for the readability & reducing the eventual error in the future maintenance.

Reply:
OK, the reason I didn't do it is because I didn't write this, I just modified it slightly, and wanted to change it as little as possible as to not increase my changed-lines count.  If I know that adding braces into old code is OK, I'll always do it (because I agree it should be done).

Reply:
If the coding standard (in contribution.md) is respected, then the code review is easier - that increases the chance of PR acceptance (slightly) indeed. But it's not the real criteria for the acceptance of PR. 

To be more clear for the acceptance of PR (not really criteria, because is quite subject):
A PR on a core function (like open/save file) which is complex (such core functions are usually complex), could be rejected more easily than the other PR, except it brings the fix for the critical bug.